.github/release.yml

```
@@ -0,0 +1,21 @@
+changelog:
+  categories:
+    - title: API change
+      labels:
+        - kind/api-change
+    - title: 🚀 Feature
+      labels:
+        - kind/feature
+    - title: 🐛 Bug or Regression
+      labels:
+        - kind/bug
+        - kind/regression
+    - title: 📖 Documentation
+      labels:
+        - kind/documentation
+    - title: Failing Test
+      labels:
+        - kind/failing-test
+    - title: 🌱 Other
```

```suggestion
    - title: 🌱 Others
```

And I would combine this with failing tests.